src/components/ChatApp/MessageSection/MessageSection.react.js

```diff
@@ -474,6 +474,10 @@ class MessageSection extends Component {
     return LoadingComponent;
   };
 
+  componentWillUnmount() {
+    document.body.style.setProperty('background-color', 'white');
```

Comment:
I didn't understand the approach. How will it work?

Reply:
@akshatnitd The problem is of higher specificity, I tried &&& to make it more specific and use styled-component, didn't work. The ChatApp styles on the body get removed if we unset background-color which solves the problem.